lms/djangoapps/certificates/management/commands/regenerate_user.py

```diff
@@ -135,7 +135,7 @@ def handle(self, *args, **options):
                 course_id
             )
 
-            if course.issue_badges:
+            if settings.FEATURES.get('ENABLE_OPENBADGES', False) and course.issue_badges:
```

Comment:
`ENABLE_OPENBADGES` is set in `lms/envs/common.py`,
so it will never be undefined.

This should be:

```
if settings.FEATURES['ENABLE_OPENBADGES']  and course.issue_badges:
    pass
```

Note: This convention is already used elsewhere for this variable; see `lms/urls.py`.